### DISCUSSION TITLE
[Question][Hw1] Error in Readme?

### DISCUSSION THREAD

**Discussion by 727yubin** [prosyslab-classroom/cs348-information-security#660] [Q&A]
Name: Yubin Lee

In the Vigenere cipher description, the decryption section has the decrypted text in uppercase. However, since it is decrypted, shouldn't it be given in lowercase?

The same goes for `test/vigenere*-dec.expected`.

**Comment by yeonhee-ryou** [comment]
Sorry for the confusion. Please follow the format of given test cases.
In vigenere decryption case, use capital letters.
